convert.py

```diff
@@ -1116,8 +1184,17 @@ def main(args_in: Optional[List[str]] = None) -> None:
     if args.dump_single:
         model_plus = lazy_load_file(args.model)
         do_dump_model(model_plus)
+        return
 
-    model_plus = load_some_model(args.model)
+    if not args.vocab_only:
+        model_plus = load_some_model(args.model)
+    else:
+        # You can no longer use guessed parameters for your vocab only model. Does anyone actually care?
```

Comment:
I feel like this is a worthwhile tradeoff. Just for example, I can check out a model via git LFS which only creates stubs for large files. Then I can generate a vocab-only model, even though the tensor data isn't there.

The downside is that when the `Params` loader falls back to guessing mode it needs the actual model data. Since right now vocab mode only loads the metadata, you can't use guessed parameters with vocab only.

If it's really important to preserve that functionality I can look at ways to get it going again.

Reply:
I think it should be good